charts/db-instances/templates/tests/test_monitoring.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if (((.Values.tests).serviceMonitor).enabled) }}
+{{- if and $.Values.tests.monitoring .Values.tests.monitoring.enabled }}
```

Comment:
You should be able to use something like
```
{{- if (.Values.tests.monitoring).enabled }}
```
instead of `and`

Reply:
Ah I was wondering about that notation. Yep it works, I will also use it for the podmonitor and servicemonitor.